test/built-ins/Temporal/ZonedDateTime/from/argument-propertybag-ignores-incorrect-properties.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.zoneddatetime.from
+description: Incorrectly-spelled properties are ignored.
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+// "1976-11-18T00:00:00+01:00[+01:00]"
+const expected = new Temporal.ZonedDateTime(217119600000000000n, "+01:00");
+
+// can be constructed with monthCode and without month
```

Comment:
This comment seems like it belongs elsewhere?

Reply:
Fixed